src/modules/villages/village_test.ts

```diff
@@ -19,27 +19,29 @@ describe('seed data', () => {
   })
 })
 
+const expectBodyFindAll = expect.arrayContaining([
+  expect.objectContaining({
+    id: expect.any(String),
+    name: expect.any(String),
+    level: expect.any(Number),
+    city: expect.any(Object),
+    category: expect.any(Object),
+    location: expect.objectContaining({
+      lat: expect.any(Number),
+      lng: expect.any(Number),
+    }),
+    images: expect.any(Array)
+  })
+])
+
 describe('tests villages', () => {
   it('test success findAll with location ', async () => {
     return request(app)
       .get('/v1/villages/list-with-location')
       .expect(200)
       .then((response) => {
         expect(response.body).toEqual(expect.objectContaining({
-          data: expect.arrayContaining([
-            expect.objectContaining({
-              id: expect.any(String),
-              name: expect.any(String),
-              level: expect.any(Number),
-              city: expect.any(Object),
-              category: expect.any(Object),
-              location: expect.objectContaining({
-                lat: expect.any(Number),
-                lng: expect.any(Number),
-              }),
-              images: expect.any(Array)
-            })
-          ]),
+          data: expectBodyFindAll,
```

Comment:
I prefer to put all the components of a test inside the test itself. Thus, I think `expectBodyFindAll` should be declared inside the `it('test success findAll ...', async () => {`.

ref: https://mtlynch.io/good-developers-bad-tests/#keep-the-reader-in-your-test-function

Reply:
by the way, the article is pretty good and can be discussed to be our test convention

Reply:
I make an error if there are more than 25 lines, next discuss test convention